.github/workflows/windows.yml

```diff
@@ -0,0 +1,108 @@
+      # - once stored under a key, they become immutable (even if local cache path content changes)
+      # - for a refresh the key has to change, e.g., hash of a tracked file in the key
+      with:
+        path: |
+          ~/.ccache
+          ~/.cache/ccache
+        key: ccache-windows-winmsvc-${{ hashFiles('.github/workflows/windows.yml') }}-${{ hashFiles('cmake/dependencies/AMReX.cmake') }}
+        restore-keys: |
+          ccache-windows-winmsvc-${{ hashFiles('.github/workflows/windows.yml') }}-
+          ccache-windows-winmsvc-
+    - name: install dependencies
+      run: |
+        Invoke-WebRequest http://fftw.org/fftw-3.3.10.tar.gz -OutFile fftw-3.3.10.tar.gz
+        7z.exe x -r fftw-3.3.10.tar.gz
+        7z.exe x -r fftw-3.3.10.tar
+        cmake -S fftw-3.3.10 -B build_fftw -DCMAKE_BUILD_TYPE=Debug
+        if(!$?) { Exit $LASTEXITCODE }
+        cmake --build build_fftw --config Debug --target install --parallel 2
+        if(!$?) { Exit $LASTEXITCODE }
+    - name: Build & Install
+      run: |
+        $env:FFTW3_DIR = "C:/Program Files (x86)/fftw/"
+        cmake -S . -B build               `
+              -DCMAKE_BUILD_TYPE=Debug    `
+              -DCMAKE_VERBOSE_MAKEFILE=ON `
+              -DHiPACE_COMPUTE=NOACC      `
+              -DHiPACE_MPI=OFF
+        if(!$?) { Exit $LASTEXITCODE }
+        cmake --build build --config Debug --parallel 2
+        if(!$?) { Exit $LASTEXITCODE }
+
+        ctest --test-dir build --config Debug --output-on-failure
+        if(!$?) { Exit $LASTEXITCODE }
+
+  build_win_clang:
+    name: Clang w/ OMP w/o MPI
+    runs-on: windows-2019
+    if: github.event.pull_request.draft == false
+    steps:
+    - uses: actions/checkout@v3
+    - name: CCache Cache
+      uses: actions/cache@v3
+      # - once stored under a key, they become immutable (even if local cache path content changes)
+      # - for a refresh the key has to change, e.g., hash of a tracked file in the key
+      with:
+        path: |
+          ~/.ccache
+          ~/.cache/ccache
+        key: ccache-windows-winclang-${{ hashFiles('.github/workflows/windows.yml') }}-${{ hashFiles('cmake/dependencies/AMReX.cmake') }}
+        restore-keys: |
+          ccache-windows-winclang-${{ hashFiles('.github/workflows/windows.yml') }}-
+          ccache-windows-winclang-
+    - uses: seanmiddleditch/gha-setup-ninja@master
+    - name: download dependencies
+      run: |
+        Invoke-WebRequest http://fftw.org/fftw-3.3.10.tar.gz -OutFile fftw-3.3.10.tar.gz
+        7z.exe x -r fftw-3.3.10.tar.gz
+        7z.exe x -r fftw-3.3.10.tar
+    - name: install dependencies
+      shell: cmd
+      run: |
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\vc\Auxiliary\build\vcvarsall.bat" x64
+        cmake -S fftw-3.3.10  ^
+              -B build_fftw   ^
+              -G "Ninja"      ^
+              -DCMAKE_C_COMPILER=clang-cl   ^
+              -DCMAKE_CXX_COMPILER=clang-cl ^
```

Comment:
Urgh, no upstream support for Clang on Windows ...
We could try to compile FFTW with MSVC and then hope for the best...
```suggestion
```